index.html

```diff
@@ -14,134 +14,150 @@
+                    </button>
+                </div>
+            </article>
+            <div class="card card4"></div>
+            <article class="card cardx">
+                <img src="images/project6.png" alt="project3" width="100%">
+                <div class="pData">
+                    <h3>Project name goes here</h3>
+                    <ul class="projectUl">
+                        <li class="projectLi">HTML/CSS</li>
+                        <li class="projectLi">Ruby on Rails</li>
+                        <li class="projectLi">Java script</li>
+                    </ul>
+                    <button class="see">
+                        See this project &nbsp →
+                    </button>
+                </div>
+            </article>
+            
+            <article class="card card5">
+                <img src="images/project4.png" alt="project4" width="100%">
+                <div class="pData">
+                    <h3>Project name goes here</h3>
+                    <ul class="projectUl">
+                        <li class="projectLi">HTML/CSS</li>
+                        <li class="projectLi">Ruby on Rails</li>
+                        <li class="projectLi">Java script</li>
+                    </ul>
+                    <button class="see">
+                        See this project &nbsp →
+                    </button>
+                </div>
+            </article>
+            <article class="card card6">
+                <img src="images/project3.png" alt="project5" width="100%">
+                <div class="pData">
+                    <h3>Project name goes here</h3>
+                    <ul class="projectUl">
+                        <li class="projectLi">HTML/CSS</li>
+                        <li class="projectLi">Ruby on Rails</li>
+                        <li class="projectLi">Java script</li>
+                    </ul>
+                    <button class="see">
+                        See this project &nbsp →
+                    </button>
+                </div>
+            </article>
+            <div class="card card7"></div>
+            <div class="card card8"></div>
+            <article class="card card9">
+                <img src="images/project5.png" alt="project6" width="100%">
+                <div class="pData">
+                    <h3>Project name goes here</h3>
+                    <ul class="projectUl">
+                        <li class="projectLi">HTML/CSS</li>
+                        <li class="projectLi">Ruby on Rails</li>
+                        <li class="projectLi">Java script</li>
+                    </ul>
+                    <button class="see">
+                        See this project &nbsp →
+                    </button>
+                </div>
+            </article>
             </div>
-        </article>
-        </div>
     </section>
 
 <!-- about section -->
 
     <section class="about">
+        <div class="about-cont">
 
+      <div class="about-head">
```

Comment:
♻️ Awesome! You are listing the tech stack correctly 🎉. However, please note the skills section doesn't resemble the Figma Template on desktop devices (< 768px)

Currently, your design looks as follows:
![image](https://user-images.githubusercontent.com/26911718/206566866-8f0c8ce6-8a0d-4e00-bf1e-9f3ad1b785b7.png)

- [ ] Please make sure you follow the original template layout 🙏. Seems like it turns into de desired layout once it hits 1015px


Figma template:
![image](https://user-images.githubusercontent.com/26911718/206567306-92df4a56-5ebb-49cd-b6c9-ec20e3339c1e.png)

Reply:
Ups... I will adjunst